epub_translator/xml/truncatable.py

```
@@ -1,16 +1,298 @@
+        from_tail: bool,
+    ) -> None:
+        """根据切割点剪枝"""
+        if not cut_point:
+            return
+
+        def prune(new_elem: Element, orig_elem: Element) -> bool:
+            """返回 True 表示找到了切割点"""
+            if orig_elem is cut_point.element:
+                # 找到切割点
+                if cut_point.location == _TextLocation.TEXT:
+                    # 切割在 text 上，删除所有子节点
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)